changelogs/unreleased/drop-net-box-console.md

```diff
@@ -1,8 +1,4 @@
 ## feature/core
 
-* **[Breaking change]** net.box console support, which was marked deprecated
+* **[Breaking change]** `net.box` console support, which was marked deprecated
   in 1.10, was dropped. Use `require('console').connect()` instead.
-
-----
-
-Breaking change: net.box console support was dropped.
```

Comment:
It is the note for a release manager. Please, keep it.

Reply:
I suggest to run `./tools/gen-release-notes` to look at the resulting release notes. In particular, it'll show such release manager notes separately to highlight them somewhere at top.